windows_containers/windows-containers-release-notes-10-15-x.adoc

```diff
@@ -28,36 +28,39 @@ For more information, see the Red{nbsp}Hat OpenShift Container Platform Life Cyc
 If you do not have this additional Red{nbsp}Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
 endif::openshift-origin[]
 
-[id="wmco-10-15-0"]
-== Release notes for Red{nbsp}Hat Windows Machine Config Operator 10.15.0
+[id="about-windows-containers-numbering"]
+== WMCO numbering
 
-This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 10.15.0 were released in link:https://access.redhat.com/errata/RHSA-2024:0954-09[RHSA-2024:0954-09].
+Starting with this release, y-stream releases of the WMCO will be in step with {product-title}, with only z-stream releases between {product-title} releases. The WMCO numbering reflects the associated {product-title} version in the y-stream position. For example, the current release of WMCO is associated with {product-title} version 4.15. Thus, the numbering is WMCO 10.15.z.
 
-=== New features and improvements
-[id="wmco-10-15-0-node-certificates"]
 
-[id="wmco-10-15-0-numbering"]
-==== New WMCO numbering
+[id="wmco-10-15-1"]
+== Release notes for Red{nbsp}Hat Windows Machine Config Operator 10.15.1
+
+This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 10.15.1 were released in link:https://access.redhat.com/errata/RHBA-2024:1191[RHBA-2024:1191].
```

Comment:
```suggestion
This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 10.15.1 were released in link:https://access.redhat.com/errata/RHBA-2024:1191[RHBA-2024:1191].
```
I assume the link will work once the errata is shipped live, yes?

Reply:
Yes. Sorry. I should have mentioned that.